contracts/gateway/tests/contract.rs

```diff
@@ -13,6 +13,8 @@ use cosmwasm_std::{
 };
 use gateway::contract::*;
```

Comment:
Maybe these should just be unit tests instead of integration tests, similar to what we do for other contracts